BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -165,7 +165,7 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
     }
 
     private func updateCurrentTheme(to newTheme: ThemeType) {
-        currentTheme = newThemeForType(newTheme)
+        currentTheme = nightModeIsOn ? newThemeForType(.dark) : newThemeForType(newTheme)
```

Comment:
Regardless of the theme argument, if the night mode is on, the dark theme is applied.